### PR TITLE
Fixes ID cards in security records console not updating properly

### PR DESCRIPTION
**Pull request by Aranclanos**
### Purpose
Fixes ID cards in security records console not updating properly

### Open Questions and Pre-Merge TODOs

- [X]  This fix is tested on the same branch it is PR'ed to.
- [X]  I correctly commented my code
- [X]  My code is indented with tabs and not spaces
- [X]  This PR does not include any unnecessary .meta, .prefab or .unity (scene) changes
- [X]  This PR does not bring up any new compile errors
- [X]  This PR has been tested in editor
- [X]  This PR has been tested in multiplayer (with 2 clients and late joiners, if applicable)

### Notes:
Basically moves the ID gameobject handling to the console rather than the GUI

### CODE REVIEW

**Comment by Jibbajabbafic** [comment]
Unsure if this fixes anything, but having the id removal and insertion on the console makes more sense, so yeah.
